testsuite/tests/singlecluster/gateway/reconciliation/change_targetref/test_update_authpolicy_target_ref.py

```diff
@@ -0,0 +1,44 @@
+"""
+Test for changing targetRef field in AuthPolicy
+"""
+
+import pytest
+
+from testsuite.kuadrant.policy.authorization.auth_policy import AuthPolicy
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.dnspolicy]
+
+
+@pytest.fixture(scope="module")
+def authorization(
+    oidc_provider, gateway, cluster, blame, module_label, route, route_b
+):  # pylint: disable=unused-argument
+    """Overwrite the authorization fixture and attach it to the gateway"""
+    policy = AuthPolicy.create_instance(cluster, blame("authz"), gateway, labels={"testRun": module_label})
+    policy.identity.add_oidc("default", oidc_provider.well_known["issuer"])
+    return policy
+
+
+def test_update_auth_policy_target_ref(
+    gateway, gateway_b, authorization, client, client_b, auth, blame, dns_policy_b, dns_policy
```

Comment:
`gateway` is already requested through the `authorization` fixture and you don't need the blame here. Test looks good :+1: